content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,163 +1,190 @@
-========================
-Sending emails with Odoo
-========================
+=======================
+Send an email with Odoo
+=======================
 
-Using your email domain in Odoo
-===============================
+Use an email domain in Odoo
+===========================
 
 Documents in Odoo (a CRM opportunity, a sales order, an invoice ...) have a discussion thread,
 called *chatter*.
```

Comment:
The content in the parenthesis is too short-form and not complete, grammatically.
As well the ellipses is not positioned.

Would do something like this instead:
```suggestion
Documents in Odoo (such as a CRM opportunity, a sales order, an invoice, etc.) have a discussion
thread, called *chatter*.
```